connectors/src/connectors/microsoft/lib/graph_api.ts

```diff
@@ -174,3 +205,17 @@ export function getDriveItemApiPath(
         parentItemApiPath.replace(/items\/[^/]+$/, `items/${item.id}`);
   return itemApiPath;
 }
+
+export function getSheetApiPath(
```

Comment:
check: important that we distinguish clearly "the whole xlsx" and "the sheets inside the xlsx"
In google it's "spreadsheets" vs "sheets". Here, is it "worksheet" vs "sheet"?
In that case, here, wouldn't it be getWorksheetApiPath?
Good to be sure that nowhere in code we refer to the full xlsx and the single sheet in the same way

Reply:
in excel it's spreadsheet and worksheet (or sometime, sheet). I'll put worksheet everywhere to be clear